### PR TITLE
refactor(rxStatus): rx-prefixes update to Status

### PR DESCRIPTION
**Pull request by cathysiller**
JIRA: https://jira.rax.io/browse/FRMW-1160

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM

### CODE REVIEW

File: src/utilities/rxStatus/scripts/rxStatus.js [hunk 1]
```diff
@@ -149,26 +149,26 @@ angular.module('encore.ui.utilities')
...
  * it is expected that almost all pages will make use of notifications, one can
- * place the repeated setup of the `Status` service in a page load event handler.
- * This will allow all pages to gain an already setup `Status` service for
+ * place the repeated setup of the `rxStatus` service in a page load event handler.
+ * This will allow all pages to gain an already setup `rxStatus` service for
  * immediate use.  For example:
  *
  * <pre>
- * .run(function ($rootScope, StatusUtil) {
+ * .run(function ($rootScope, rxStatusUtil) {
```
Comment (CITguy) [inline src/utilities/rxStatus/scripts/rxStatus.js]: This PR should not prefix `StatusUtil`, please revert to previous value.

File: src/utilities/rxStatus/scripts/rxStatus.js [hunk 2]
```diff
@@ -149,26 +149,26 @@ angular.module('encore.ui.utilities')
...
  *     $rootScope.$on('$routeChangeSuccess', function () {
- *         Status.setupScope($rootScope);
+ *         rxStatus.setupScope($rootScope);
```
Comment (CITguy) [inline src/utilities/rxStatus/scripts/rxStatus.js]: This should match the injectable in the `.run()` function.

```javascript
StatusUtil.setupScope($rootScope);
```

File: src/utilities/rxStatus/scripts/rxStatus.js [hunk 3]
```diff
@@ -256,7 +256,7 @@ angular.module('encore.ui.utilities')
         scope.loaded = false;
     };
 
-    status.setStatus = function (msg, state) {
+    status.setrxStatus = function (msg, state) {
```
Comment (CITguy) [inline src/utilities/rxStatus/scripts/rxStatus.js]: Please revert `status.setrxStatus` back to `status.setStatus`. This is considered a breaking change to the API.

File: src/utilities/rxStatus/scripts/rxStatus.spec.js [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        module('encore.ui.utilities');
+
+        inject(function ($rootScope, rxStatus) {
+            scope = $rootScope.$new();
+            status = rxStatus;
+            rootScope = $rootScope;
+
+            status.setScope(scope);
+
+            sinon.spy(status, 'setrxStatus');
```
Comment (CITguy) [inline src/utilities/rxStatus/scripts/rxStatus.spec.js]: Please revert `setrxStatus` back to `setStatus` in this file.

File: utils/rx-page-objects/src/rxSelectFilter.page.js [hunk 5]
```diff
@@ -44,7 +44,7 @@ var rxSelectFilter = {
      *     });
      *
      *     expect(myPage.myTable.column('Account').data.then(_.uniq)).to.eventually.eql(['US']);
-     *     expect(myPage.myTable.column('Status').data.then(_.uniq)).to.eventually.eql(['Open']);
+     *     expect(myPage.myTable.column('rxStatus').data.then(_.uniq)).to.eventually.eql(['Open']);
```
Comment (CITguy) [inline utils/rx-page-objects/src/rxSelectFilter.page.js]: Please revert. This is the name of a table column, not the `Status` service.

File: utils/rx-page-objects/src/rxStatusColumn.page.js [hunk 6]
```diff
@@ -14,7 +14,7 @@ var classNameToStatus = function (iconClassName) {
  * @namespace
  * @description Functionality for interacting with and query the values in a status column. When
  * used in conjunction with {@link rxSortableColumn}, these functions will allow you to interact
- * with a single cell in that sortable column as a "status column". Status columns have use a mix
+ * with a single cell in that sortable column as a "status column". rxStatus columns have use a mix
```
Comment (CITguy) [inline utils/rx-page-objects/src/rxStatusColumn.page.js]: Please revert.

File: utils/rx-page-objects/src/rxStatusColumn.page.js [hunk 7]
```diff
@@ -158,7 +158,7 @@ exports.rxStatusColumn = {
 
     /**
      * @function
-     * @param {ElementFinder} rxStatusCellElement - Status cell element from a table row.
+     * @param {ElementFinder} rxStatusCellElement - rxStatus cell element from a table row.
```
Comment (CITguy) [inline utils/rx-page-objects/src/rxStatusColumn.page.js]: Please revert.

File: utils/rx-page-objects/test/rxMetadata.midway.js [hunk 8]
```diff
@@ -51,7 +51,7 @@ describe('rxMetadata', function () {
         demoPage.go('#/elements/Metadata');
     });
 
-    describe('Status', encore.exercise.rxMetadata({
+    describe('rxStatus', encore.exercise.rxMetadata({
```
Comment (CITguy) [inline utils/rx-page-objects/test/rxMetadata.midway.js]: Please revert.

File: utils/rx-page-objects/test/rxStatusColumn.midway.js [hunk 9]
```diff
@@ -22,8 +22,8 @@ var tablePageObject = Page.create({
 
     column: {
         value: function (columnName) {
-            // Only 'Status' is supported -- 'Title' is unsortable.
-            if (columnName === 'Status') {
+            // Only 'rxStatus' is supported -- 'Title' is unsortable.
+            if (columnName === 'rxStatus') {
```
Comment (CITguy) [inline utils/rx-page-objects/test/rxStatusColumn.midway.js]: Please revert. This is a name of a column, not a service.

File: utils/rx-page-objects/test/rxStatusColumn.midway.js [hunk 10]
```diff
@@ -246,7 +246,7 @@ describe('rxStatusColumn', function () {
         };
 
         before(function () {
-            column = tablePageObject.column('Status');
+            column = tablePageObject.column('rxStatus');
```
Comment (CITguy) [inline utils/rx-page-objects/test/rxStatusColumn.midway.js]: Please revert. This is a name of a column, not a service.

File: src/utilities/rxStatus/scripts/rxStatus.js (hunk 1 above)
Comment (cathysiller) [inline src/utilities/rxStatus/scripts/rxStatus.js]: reverted

File: src/utilities/rxStatus/scripts/rxStatus.js (hunk 2 above)
Comment (cathysiller) [inline src/utilities/rxStatus/scripts/rxStatus.js]: updated

File: src/utilities/rxStatus/scripts/rxStatus.js (hunk 3 above)
Comment (cathysiller) [inline src/utilities/rxStatus/scripts/rxStatus.js]: reverted

File: src/utilities/rxStatus/scripts/rxStatus.spec.js (hunk 4 above)
Comment (cathysiller) [inline src/utilities/rxStatus/scripts/rxStatus.spec.js]: reverted

File: utils/rx-page-objects/src/rxSelectFilter.page.js (hunk 5 above)
Comment (cathysiller) [inline utils/rx-page-objects/src/rxSelectFilter.page.js]: reverted

File: utils/rx-page-objects/src/rxStatusColumn.page.js (hunk 7 above)
Comment (cathysiller) [inline utils/rx-page-objects/src/rxStatusColumn.page.js]: reverted

File: utils/rx-page-objects/test/rxMetadata.midway.js (hunk 8 above)
Comment (cathysiller) [inline utils/rx-page-objects/test/rxMetadata.midway.js]: reverted

File: utils/rx-page-objects/test/rxStatusColumn.midway.js (hunk 9 above)
Comment (cathysiller) [inline utils/rx-page-objects/test/rxStatusColumn.midway.js]: reverted

File: utils/rx-page-objects/test/rxStatusColumn.midway.js (hunk 10 above)
Comment (cathysiller) [inline utils/rx-page-objects/test/rxStatusColumn.midway.js]: reverted

File: utils/rx-page-objects/src/rxStatusColumn.page.js (hunk 6 above)
Comment (cathysiller) [inline utils/rx-page-objects/src/rxStatusColumn.page.js]: reverted

File: src/utilities/rxStatus/scripts/rxStatus.js [hunk 11]
```diff
@@ -149,26 +149,26 @@ angular.module('encore.ui.utilities')
...
- * This will allow all pages to gain an already setup `Status` service for
+ * place the repeated setup of the `rxStatus` service in a page load event handler.
+ * This will allow all pages to gain an already setup `rxStatus` service for
  * immediate use.  For example:
  *
  * <pre>
  * .run(function ($rootScope, StatusUtil) {
  *     $rootScope.$on('$routeChangeSuccess', function () {
- *         Status.setupScope($rootScope);
+ *        StatusUtil.setupScope($rootScope);
```
Comment (CITguy) [inline src/utilities/rxStatus/scripts/rxStatus.js]: Missing one space in indent.

File: src/utilities/rxStatus/scripts/rxStatus.spec.js [hunk 12]
```diff
@@ -1,12 +1,12 @@
-describe('utilities: Status', function () {
+describe('utilities: rxStatus', function () {
```
Comment (CITguy) [inline src/utilities/rxStatus/scripts/rxStatus.spec.js]: Please add a `TODO` comment to this file to refactor these tests.

File: src/utilities/rxStatus/scripts/rxStatus.js (hunk 11 above)
Comment (cathysiller) [inline src/utilities/rxStatus/scripts/rxStatus.js]: thumbs up

File: src/utilities/rxStatus/scripts/rxStatus.spec.js (hunk 12 above)
Comment (cathysiller) [inline src/utilities/rxStatus/scripts/rxStatus.spec.js]: thumbs up

**Comment by lentusignavus** [comment]
DEV LGTM

**Pull request merged**